apdx-validation.tex

```diff
@@ -726,7 +726,7 @@ \subsubsection*{Rules for the \class{Participation} class}
 \printWarning{All \sbol{URI}s contained by the \sbolmult{roles:P}{roles} property of an \sbol{Participation} MUST refer to non-conflicting ontology terms.\\ Reference: \sec{sec:Participation}}
 
 \twozeroone{
-\printModeling{Exactly one role in the set of \sbolmult{roles:P}{roles} SHOULD be a \sbol{URI} from the participant role branch of the SBO.\\ Reference: \sec{sec:Participation}}
+\printModeling{Exactly one role in the set of \sbolmult{roles:P}{roles} SHOULD be a \sbol{URI} in \ref{tbl:participant_roles}.\\ Reference: \sec{sec:Participation}}
```

Comment:
Why are we losing the reference to Systems Biology Ontology?

Reply:
The participant_roles table now being referenced shows that the URI is for SBO terms, should we make it explicit here as well?

Reply:
We refer to SBO above with interactions.  There's no harm in having local references to make it easier for people to understand.

Reply:
would 
```
\printModeling{Exactly one role in the set of \sbolmult{roles:P}{roles} SHOULD be a \sbol{URI} from the participant role branch of the SBO (see \ref{tbl:participant_roles}) .\\ Reference: \sec{sec:Participation}}
```
be a more appropriate update?

Reply:
That works as far as I'm concerned.

Reply:
updated-- there was an accidental merge before I could make the change, so I opened a new MR (#207) and added you as a reviewer there